tools/mongo-access.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+  
+machine=${1:-0}
+model=${2:-controller}
+
```

Comment:
Would be nice to have some basic parameter handling and help, just so when people run the first time don't need to open the script to understand the parameters.

Reply:
I agree, but I would also be ok with doing that in a follow-up PR.